File: sdks/java/basys.sdk/src/org/eclipse/basyx/aas/metamodel/hashmap/aas/enums/SchemaType.java
package org.eclipse.basyx.aas.metamodel.hashmap.aas.enums;

public enum SchemaType {

	None(0),
	XSD(1),
	RDFS(2),
	JSchema(3);
	
	private int id;

	SchemaType(int id) {
        this.id = id;
    }

    public int getId() { 
        return id;
    }
}

File: sdks/csnet/BaSys40.Models/Core/AssetAdministrationShell/Enums/SchemaType.cs
using System.Runtime.Serialization;

namespace BaSys40.Models.Core.AssetAdministrationShell.Enums
{
    [DataContract]
    public enum SchemaType : int
    {
        None = 0,
        XSD = 1,
        RDFS = 2,
        JSchema = 3
    }
}
